### DISCUSSION TITLE
100 Tick HL2DM?

### DISCUSSION THREAD

**Discussion by ViperbirdHL2DM** [GameServerManagers/LinuxGSM#4062] [General]
Hi,

I am new to all of this, but haven't been able to find an answer or get 100 tick working myself using Linux GSM for Half-Life 2 Deathmatch.

I have tried both the 'hl2dmserver.cfg' and the server.cfg files with:
sv_mincmdrate 100
sv_maxcmdrate 101
sv_minrate 100000
sv_maxrate 0
sv_minupdaterate 100
sv_maxupdaterate 101

I have also tried adding in the cfg files /lgsm/config-lgsm/hl2dmserver
tickrate="100"
and also
-tickrate 100
with no luck

Any suggestions on how to get 100 Tick working?

**Comment by Xeogin** [comment]
Just found out how to set this and have it apply. The documentation for it is here, but it doesn't explain it particularly well: https://docs.linuxgsm.com/configuration/start-parameters

What you need to do is open the hl2dmserver.cfg file like you were and add the following instead:
`startparameters="-game hl2mp -strictportbind -ip ${ip} -port ${port} +clientport ${clientport} +tv_port ${sourcetvport} +map ${defaultmap} +servercfgfile ${servercfg} -maxplayers ${maxplayers} -tickrate 100"`
